rsmtool/preprocessor.py

```diff
@@ -479,18 +508,17 @@ def remove_outliers_using_truncations(self, values, feature_name, truncations):
 
```

Comment:
Missing some info in the docstring: IIUC another difference from `remove_outliers` is that it processes just one feature? 
and it is assumed that the given data is only values of `feature_name`?